### PR TITLE
osc 1.0.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by Porkepix** [comment]
Why aren't the python resources updated automatically? Here for `chardet`?

**Comment by chenrui333** [comment]
> Why aren't the python resources updated automatically? Here for `chardet`?

we only resolve pypi resource.

**Comment by Porkepix** [comment]
> > Why aren't the python resources updated automatically? Here for `chardet`?
> 
> we only resolve pypi resource.

But pythonhosted.org links are PyPI ones, right?

**Comment by chenrui333** [comment]
> > > Why aren't the python resources updated automatically? Here for `chardet`?
> > 
> > 
> > we only resolve pypi resource.
> 
> But pythonhosted.org links are PyPI ones, right?

yeah, it is using pypi source to resolve the dependencies, that is why it happened, you can try out the old version like `0.182.0` and run `brew update-python-resources osc` to find out.

File: Formula/osc.rb [hunk 1]
```diff
@@ -20,57 +20,53 @@ class Osc < Formula
...
   depends_on "python@3.11"
 
   uses_from_macos "curl"
 
-  resource "chardet" do
-    url "https://files.pythonhosted.org/packages/31/a2/12c090713b3d0e141f367236d3a8bdc3e5fca0d83ff3647af4892c16c205/chardet-5.0.0.tar.gz"
-    sha256 "0368df2bfd78b5fc20572bb4e9bb7fb53e2c094f60ae9993339e8671d0afb8aa"
+  resource "cffi" do
+    url "https://files.pythonhosted.org/packages/2b/a8/050ab4f0c3d4c1b8aaa805f70e26e84d0e27004907c5b8ecc1d31815f92a/cffi-1.15.1.tar.gz"
+    sha256 "d400bfb9a37b1351253cb402671cea7e89bdecc294e8016a707f6d1d8ac934f9"
```
Comment (carlocab) [inline Formula/osc.rb:32]: You can also do
```ruby
depends_on "cffi"
```

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**